docs/installation.md

```diff
@@ -41,9 +41,12 @@ services:
     ports:
       - 8080:80
     environment:
-      - GOTIFY_DEFAULTUSER_PASS=custom
+      GOTIFY_DEFAULTUSER_PASS: 'my-com"plex.\password'
```

Comment:
We have a lot of issues in gotify/server about users which cannot login which is due to this password change. It's on my todo list to change this to `admin` so that the docs.
https://gotify.net/docs/first-login match this default.

I actually rather have this be `admin` in this example. What do you think?